src/extension.ts

```diff
@@ -122,16 +118,16 @@ function creatDataVirtView(): void {
 	});
 	dataVirtTreeView.onDidChangeVisibility(async () => {
 		if (dataVirtTreeView.visible === true) {
-			await dataVirtProvider.refresh().catch(err => console.log(err));
+			await dataVirtProvider.refresh().catch((err) => console.log(err));
```

Comment:
paranthesis are overcomplicated? I am not going to agree here. Its a style setting I enabled for this project to always have these paranthesis. Unless we decide at some point to not do that for all of our projects I will keep it for now.

Reply:
ok, this is the default for ESLint https://eslint.org/docs/rules/arrow-parens

I wasn't aware of the ambiguity that can be triggered https://eslint.org/docs/rules/arrow-parens#if-statements

it would have been better to do the modification in a specific PR.